docs/language.rst

```diff
@@ -125,14 +125,18 @@ Literal values can be assigned to variables
 Primitive types
 ==============================
 
-The basic primitive types are ``string``, ``number``, ``int`` or ``bool``. These basic types also support type casts:
+The basic primitive types are ``string``, ``number``, ``float``, ``int`` or ``bool``. These basic types also support type casts:
+
+.. note::
+    ``number`` will be deprecated in the future
```

Comment:
It is already deprecated in this PR, isn't it? I would not mention deprecated types in the list above.